core/src/test/java/feast/core/job/dataflow/DataflowRunnerConfigTest.java

```diff
@@ -38,10 +38,12 @@ public void shouldConvertToPipelineArgs() throws IllegalAccessException {
             .setNetwork("default")
             .setSubnetwork("regions/asia-east1/subnetworks/mysubnetwork")
             .setMaxNumWorkers(1)
+            .setServiceAccount("serviceaccount")
```

Comment:
Why is service account being modified here? Is this a mistake?

Reply:
Service account is part of the available configurations, but it was never tested in the test. We can have this in another PR is it is preferable.

Reply:
Its still not clear if its necessary or not.

Reply:
I have removed the service account argument from the test.